resource-definitions/empty_test.go

```diff
@@ -0,0 +1,6 @@
+// This is an empty test file used to force Go test to include this package in
+// coverage reports even though there are no existing (non-empty) Go test
+// files. If/when Go tests are added for this package, please delete this
+// empty test file since it will no longer be needed.
+
+package main
```

Comment:
I'm not sure we need this file. The other .go files in this directory have the `//go:build ignore` tag b/c they're just samples.

Reply:
Ah, good catch!